### ISSUE TITLE
Writing to the database fails silently when writing tuple with no Exception raised and no effect on the database

### ISSUE THREAD

**Issue by lukas-ba-github** [influxdata/influxdb-client-python#150]
I spent a bit of time wondering why influxdb does not contain my data or why there is no error raised by influxdb-client-python so please consider this bug report.

__Steps to reproduce:__
```
import datetime
from influxdb_client import InfluxDBClient, Point, WritePrecision
from influxdb_client.client.write_api import SYNCHRONOUS

dT = datetime.datetime.utcnow()

org = "testOrg"
url = "http://localhost:8086"
token = '-'
bucket = "testBucket"

p = (
    Point("testM")
    .time(dT, write_precision=WritePrecision.S)
    .tag("testT", "abc")
    .field("testF", 1)
)
client = InfluxDBClient(url=url, token=token, org=org)
write_api = client.write_api(write_options=SYNCHRONOUS)
# fails silently, no error raised even if the database does not exist
record = (p, )
write_api.write(bucket, org, record)

# works
#record = [p, ]
#write_api.write(bucket, org, record)
```

__Expected behavior:__
Option A:
- a point is inserted into the database
-  Tuples and other iterables are accepted instead of Lists to be more pythonic

Option B:
- only Lists are accepted
- a ValueError is raised for Tuples

__Actual behavior:__
Nothing happens

__Specifications:__
 - Client Version: influxdb-client 1.10.0
 - InfluxDB Version: influxdb 1.8.2-1
 - Platform: Linux, Arch Linux

**Comment by bednar** [comment]
Hi @lukas-ba-github,

thanks for using our client.

It is a nice catch. The option A seems to be a better solution so we will fix it as you suggested.

Regards

**Comment by bednar** [comment]
Hi @lukas-ba-github,

The issue is fixed in [1.11.0 milestone](https://github.com/influxdata/influxdb-client-python/milestone/13). 

If you would like to use a dev version then install client via:

```pip install git+https://github.com/influxdata/influxdb-client-python.git@master```

Regards
